README.md

```diff
@@ -2,7 +2,21 @@
 
```

Comment:
The commit message is too vague. I understand that sometimes we can't summarize the change in one short line. But in that case you have two options:

- Add more text after the header describing the change (e.g: "Add notes on supported browsers and fix the example code"
- Maybe the problem is that you are mixing changes that should be different commits. I think that's the case here. This should have been two separate commits: "Add supported browsers to the README" and "Fix example code  (we don't expose HttpUtil now)".

If you can run the rebase trick to split this in two commits it would be ideal. Otherwise let's go ahead but avoid doing it again.

Reply:
I have split the commit in two.